lib/PuppeteerSharp.Tests/PageTests/CloseTests.cs

```diff
@@ -24,11 +24,21 @@ public async Task ShouldRejectAllPromisesWhenPageIsClosed()
             Assert.Equal("Target.detachedFromTarget", exception.CloseReason);
         }
 
-        [Fact]
-        public async Task ShouldNotBeVisibleInBrowserPages()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
```

Comment:
Apologies if this was not what you were going for with your suggestions on https://github.com/hardkoded/puppeteer-sharp/issues/1456#issuecomment-610633676. I wasn't sure and figured that it would be easier when looking at the code.